not-supported/daily-backup-wizard.sh

```diff
@@ -47,16 +55,38 @@ Please rename or delete $BACKUP_SCRIPT_NAME if you want to reconfigure the backu
     exit 1
 fi
 # Check if pending snapshot is existing and cancel the setup in this case.
+if does_snapshot_exist "NcVM-startup"
+then
+    # Cannot get executed during the startup script
+    if [ -f "$SCRIPTS/nextcloud-startup-script.sh" ]
+    then
+        msg_box "The daily backup cannot get configured during the startup script.
+Please try again after it is finished by running: 
+'sudo bash $SCRIPTS/menu.sh' -> 'Server Configuration' -> 'Daily Backup Wizard'."
+        exit
+    fi
+    msg_box "You need to run the update script once before you can continue with creating the backup script."
+    if yesno_box_yes "Do you want to do this now?"
+    then
+        bash "$SCRIPTS"/update.sh minor
```

Comment:
Better instruct to use the menu script since the update script may change.

Reply:
hm... but in this case we can simply change the command ourselves.
(I don't recommend to do it like this to the users but instead execute this command directly for the sake of simplicity.)

Reply:
Seems to me that the minor switch doesn't work for some reason, but that's for another PR.

Reply:
Hm... I think the minor switch worked in my testing but yes, that's definitely something for a different PR.